src/utils/breakdown_validation.py

```diff
@@ -0,0 +1,122 @@
+"""Function to validate the breakdown totals from the imputation."""
+import os
+import logging
+import pandas as pd
+import numpy as np
+from pandas import DataFrame as pandasDF
+
+
+BreakdownValidationLogger = logging.getLogger(__name__)
+
+"""Function to validate the breakdown totals from the imputation."""
+def breakdown_validation(df: pd.DataFrame) -> dict:
+    """Function to check that the breakdown values match the criteria provided. 
+    
+    Args:
+        df - the BERD 2023 form dataframe that is to be checked
+    Returns:
+        A boolean dictionary and messages which update based upon if there are any
+        issues with the logic of the columns.   
+    
+    """
+    bool_dict = {}
+    msg = ""
+
+    df.loc[df['203'] == 0, ['222', '223']] = 0
```

Comment:
As discussed, let's move this to a separate function and call at the beginning of `run_breakdown_validation()` - it will also need its own test